src/sage/arith/misc.py

```diff
@@ -3600,6 +3600,13 @@ def CRT_list(values, moduli=None):
         [1, 2, 3]
         sage: ms
         [5, 7, 9]
+    
+    Tests for call with length 1 lists::
```

Comment:
```suggestion
    Tests for call with length 1 lists (:issue:`40074`)::
```
Good to reference the explicit PR (or the base issue). Otherwise LGTM.